README.markdown

```diff
@@ -51,13 +52,22 @@ Add the class to agents you want to upgrade:
 include ::puppet_agent
 ~~~
 
+It will install the latest released version of Puppet from Puppet Collection 1.
+
+The recommended way to upgrade with this module is to first standup a Puppet Server 2.1 master - which supports backward compatibility with Puppet 3 agents - and point the agent to upgrade at it. Once you've confirmed the agent runs successfully against the new master, and thus the Puppet 4 language, apply the class to the agent and confirm that it checks back in after a successful upgrade. Further details on upgrading are available in Puppet's documentation.
```

Comment:
Further details I'm mainly referring to https://github.com/puppetlabs/puppet-docs/blob/july2015_improved_upgrade/source/puppet/4.2/reference/upgrade_major_agent.markdown, which is still unreleased. The eventual URL for that will probably be linked from http://docs.puppetlabs.com/puppet/latest/reference/#upgrading-from-puppet-3x, but I'm not sure of the final link so I'm wary to put it in the module.